webapp/components/use/imageSegmentation.ts

```diff
@@ -78,33 +78,56 @@ export class VirtualBackgroundStream {
     // 3 - face-skin
     // 4 - clothes
     // 5 - others (accessories)
-    const maskData = segmentationResult.categoryMask.getAsFloat32Array()
-
-    for (let i = 0; i < maskData.length; ++i) {
-      const maskVal = maskData[i]
-      const j = i * 4
-      // set chosen pixels to transparent
-      if (maskVal == 0) {
-        imageData[j + 3] = 0 // alpha channel
+    const maskData = segmentationResult.categoryMask.getAsFloat32Array().map(val => Math.round(val * 255.0))
+    const categoryChosen = 0
+
+    // draw the top layer on tempCanvas
+    {
+      for (let i = 0; i < maskData.length; i++) {
+        if (maskData[i] != categoryChosen) {
+          maskData[i] = 255
+        } else {
+          maskData[i] = 0
+        }
```

Comment:
```suggestion
        maskData[i] = maskData[i] === categoryChosen ? 0 : 255
```

Can be simplified, and use strict equals `===` or `!==`